server/src/models/mod.rs

```diff
@@ -394,6 +395,26 @@ mod tests {
         assert_eq!(10, selected_model.values.len());
     }
 
+    #[test]
+    fn test_selected_model_new() {
```

Comment:
Maybe change the name of this test to better describe what exactly it is testing.

Reply:
Maybe the new name is even more convoluted than before. If so, do you have any suggestions?

Reply:
Maybe something like `test_model_with_fewest_bytes_is_selected"

Reply:
Sure, fixed.